### PR TITLE
[1210] Add ucas_status to courses table

### PR DESCRIPTION
**Pull request by tvararu**
:shipit: :shipit: :shipit:

### CODE REVIEW

**Comment by tvararu** [comment]
[EPIC: Courses table in manage-courses-frontend](https://trello.com/c/Ju2Kfadq/1210-epic-courses-table-in-manage-courses-frontend)

File: app/helpers/course_helper.rb [hunk 1]
```diff
@@ -20,4 +20,12 @@ def course_content_tag_css_class(course)
       "published_with_unpublished_changes" => "phase-tag--published"
     }[course.attributes[:content_status]]
   end
+
+  def course_ucas_status(course)
+    {
+      "running" => "Running",
+      "new" => "New – not yet running",
+      "not_running" => "Not running"
+    }[course.attributes[:ucas_status]]
```
Comment (misaka) [inline app/helpers/course_helper.rb]: Why `course.attributes`? Shouldn't `course.ucas_status` work?

Comment (tvararu) [reply app/helpers/course_helper.rb]: ¯\\\_(ツ)\_\/¯ will try

Comment (tvararu) [reply app/helpers/course_helper.rb]: Tests blew up:

```
Failures:

  1) View helpers #course_ucas_status returns correct content
     Failure/Error: }[course.ucas_status]

     NoMethodError:
       undefined method `ucas_status' for #<JSONAPIMockSerializable:0x00007ff11645be10>
     # ./app/helpers/course_helper.rb:29:in `course_ucas_status'
     # ./spec/helpers/course_helper_spec.rb:34:in `block (3 levels) in <top (required)>'
```

**Pull request merged**